travis-ci/vmtest/run.sh

```diff
@@ -243,7 +243,7 @@ create_rootfs_img() {
 download_rootfs() {
 	local rootfsversion="$1"
 	local dir="$2"
-	download "${PROJECT_NAME}-vmtest-rootfs-$rootfsversion.tar.zst" |
+	download "$ARCH/${PROJECT_NAME}-vmtest-rootfs-$rootfsversion.tar.zst" |
```

Comment:
nit: `${ARCH}` for consistency?

Reply:
OK (I don't think `{}` does anything useful here, but I agree that it will look better).